internal/ethapi/api.go

```diff
@@ -1656,6 +1656,10 @@ func (s *PublicTransactionPoolAPI) SendRawTransaction(ctx context.Context, encod
 	if err := rlp.DecodeBytes(encodedTx, tx); err != nil {
 		return common.Hash{}, err
 	}
```

Comment:
This is missing the check for `eth_sendRawEthSignTransaction`, the check could either be duplicated or moved deeper into the stack, ie  into`SubmitTransaction`

Reply:
Oh whoops great catch -- adding